### DISCUSSION TITLE
redux in react

### DISCUSSION THREAD

**Discussion by icodejsx** [ken286/WeWe#1] [Q&A] [answered]
how to i  understand redux in react

**Comment by DonGuillotine** [comment, marked as answer]
Understanding Redux in React can be complicated and there is a lot to learn. The best way to get started is to read through some tutorials and practice coding examples. There are many online tutorials available that can provide step-by-step guidance on how to use Redux within React. You can also find YouTube videos and other resources that can help you understand the basics.
